### ISSUE TITLE
Updates...? You've removed your QBMenu and QBinput and deprecates.

### ISSUE THREAD

**Issue by TristGit** [ohbrime/qbcore-fivem-appearance#9]
What you dont seem to mention ANYWHERE is your custom implementation of a PD outfits menu. Where did the support go?

**Comment by heyjeay** [comment]
Yeah i noticed, created a new server and now nothing seems to work with the current version and menus
